protocol-designer/src/components/modals/CreateFileWizard/utils.ts

```diff
@@ -60,28 +59,32 @@ export const getLastCheckedEquipment = (
 }
 
 export const getCrashableModuleSelected = (
-  modules: FormModulesByType,
+  modules: FormModules | null,
   moduleType: ModuleType
 ): boolean => {
-  const formModule = modules[moduleType]
+  if (modules == null) return false
```

Comment:
from type, this won't be `undefined`. 

```suggestion
  if (modules === null) return false
```